happiness-backend/tests/test_happiness.py

```diff
@@ -427,3 +406,109 @@ def test_discussion_comments(init_client):
     comments = comment_access_u3.json
     assert len(comments) == 2
     assert comments[0]['author']['id'] == 1 and comments[1]['author']['id'] == 3
+
+
+def test_happiness_search(init_client):
```

Comment:
Just as you did for Happiness, split this into separate, smaller unit test cases. For example one for searching with value, one for searching with dates, one for both, etc.. This way if the test fails it will be easier to know which is going wrong.

Reply:
makes sense but i have one concern which is for all of my tests on search they currently all search within a set of happiness entries that i created with a bunch of post requests from lines 413 to line 453, basically establishing a bunch of entries for me to test filtering through. If i seperate all the searches into multiple test cases, will i need to copy these lines into all of those test cases? i assume there is a better way than to do that, maybe I could call those test cases within one larger test case, but if i did that would the program treat those calls as test cases or just helper functions?